src/crewai/utilities/events/crewai_event_bus.py

```diff
@@ -1,14 +1,14 @@
 import threading
 from contextlib import contextmanager
-from typing import Any, Callable, Dict, List, Type, TypeVar, cast
+from typing import Any, Callable, Dict, Final, List, Type, TypeVar, cast
 
 from blinker import Signal
 
 from crewai.utilities.events.base_events import CrewEvent
 from crewai.utilities.events.event_types import EventTypes
 
 EventT = TypeVar("EventT", bound=CrewEvent)
-
+WILDCARD: Final = CrewEvent(type="wildcard")
```

Comment:
This was to maintain type compatibility with the rest of the code. Suggestions welcome!